src/frontend/screens/AppPasscode/InputPasscode.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+import { StyleSheet, View } from "react-native";
+import { useBlurOnFulfill } from "react-native-confirmation-code-field";
+
+import { PasscodeScreens } from ".";
+import { useNavigationFromRoot } from "../../hooks/useNavigationWithTypes";
+
+import { RED, WHITE } from "../../lib/styles";
+import Button from "../../sharedComponents/Button";
+import {
+  CELL_COUNT,
+  PasswordInput,
+} from "../../sharedComponents/PasscodeInput";
+import Text from "../../sharedComponents/Text";
+
+const m = defineMessages({
```

Comment:
thinkk we should be consistent with the wording and stick to "passcode" for everything, as opposed to "password"

Reply:
Lets wait...we are submitting the copy deck to the programs team next week....so lets finalize that first before we change any of the language